src/main/resources/application.yml

```
@@ -1,9 +1,36 @@
 spring:
   application:
     name: onem-backend-java
+  profiles:
+    active: dev
 
 shortener :
   expiredMinute : 1
   blockedDomains :
     - "https://mvnrepository.com"
-    - "https://papago.naver.com"
\ No newline at end of file
+    - "https://papago.naver.com"
+
+---
+spring:
+  config:
+    activate:
+      on-profile: dev  # 개발 환경(dev) 설정
```

```suggestion
spring.config.activate,on-profile: dev  # 개발 환경(dev) 설정
```

`nit` 저는 local profile 을 분리하는 편 입니다.
- 그리고 본 파일에서는 dev prorilfe 이 2번 나오고 있습니다. override 의 위험이 있습니다.